dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -515,7 +517,7 @@ private Response getFileByToolVersionID(String registryId, String versionIdParam
                 if (potentialDockerfile.isPresent()) {
                     ExtendedFileWrapper dockerfile = new ExtendedFileWrapper();
                     //TODO: hook up file checksum here
-                    dockerfile.setChecksum(Lists.newArrayList());
+                    dockerfile.setChecksum(getFileDescriptorChecksumsForTrs(potentialDockerfile.get()));
```

Comment:
I haven't implemented calculating the checksum for tool dockerfiles/descriptor files. I hooked this up in advance since I was already in the area, but it should continue returning an empty arraylist for now. As for whether we should or not, I'm not sure. I supposed @denis-yuen wanted it because he left the to-do, but maybe it was accidental.

Reply:
I don't see much harm in getting checksums for everything and sorting it out later.